### ISSUE TITLE
Host entity names inconsistent in Satellite and registry

### ISSUE THREAD

**Issue by chbrown13** [SatelliteQE/satellite-populate#83]
* satellite-populate version:
* Python version: 2.7.13
* Operating System: Fedora 25

### Description

Satellite automatically modifies the name of Host entities which causes problems for searching and performing actions for satellite-populate. Satellite automatically stores host names and appends a domain but the satellite-populate registry still refers to the object by the original name.

Example:
* **satellite-populate original name**
```
vars:
  host_name: Update Host
# Create...
```
* **Satellite name** 
```
updatehost.8redwimjy5
```

**Comment by rochacbruno** [comment]
That is new to me, I did not know satellite changed the names. So in `populate` the search should be done by other atributes. This will be very hard to fix.

**Comment by chbrown13** [comment]
Agreed, I also had to set ```_force_raw``` to True in constants.py because searching Host entities was running into an issue returning all hosts instead of the one specified, same as #66. So couldn't search with other attributes.
